tools/clang/test/clang-hlsl/lit.cfg

```diff
@@ -35,3 +35,6 @@ test_dir = os.path.join(config.llvm_obj_root, config.build_mode, 'test')
 #projects\dxilconv\test
 config.test_format = lit.formats.TaefTest(te, test_dll, hlsl_data_dir, test_dir)
 
+# TAEF only runs on Windows
+if platform.system() != 'Windows':
+  config.unsupported = True
```

Comment:
Is this necessary? Won't this get ignored per the changes to the above file?

Will we be using this config to run other tests through lit on non-windows platforms going forward even though, as I read it, all it does for now is TAEF?

Reply:
The CMake change prevents generating the target to run the lit suite, this change marks the suite unsupported so that if the suite is run along other suites (as check-clang or check-all will hopefully work), it is still disabled.